docker/checkout.sh

```diff
@@ -25,6 +25,6 @@ if [[ -n ${1} ]]; then
 fi
 
 cd ${dir}/src
-git clone --depth=1 https://github.com/usdot-fhwa-stol/CARMAMsgs.git --branch develop
+git clone --depth=1 https://github.com/usdot-fhwa-stol/CARMAMsgs.git --branch feature/add_plugin_type
```

Comment:
This should be shifted back to develop before merge, or I suppose really changes to this file shouldn't be commited most of the time, only edited locally.

Reply:
Good catch. I made this change to build docker image. I will change it back.